### PR TITLE
add formatting requirement to squlalchemy uri option

### LINKED ISSUE #502: Document `SQLALCHEMY_DATABASE_URI` Configuration Option
### Docs Update Description

Instead of providing the individual elements of the Postrges database as configuration, users can instead provide `FIDESOPS__DATABASE__SQLALCHEMY_DATABASE_URI`, which allows them to use parameters like `sslmode` etc. 

This change should be made for this page: https://ethyca.github.io/fidesops/guides/configuration_reference/#

This connection string should be in the format of `postgresql://<user>:<password>@<hostname>/<database>`

### Additional context

This is required for Azure deployments that use Azure database for postgresql, which requires `sslmode=required`.

Aha! Link: https://ethyca.aha.io/features/OPS-59

### PR DESCRIPTION
**Pull request by conceptualshark**
# Purpose

* Quick fix to add another line to the `FIDESOPS__DATABASE__SQLALCHEMY_DATABASE_URI` config option to provide a sample connection string.

# Checklist
- [x] Applicable documentation updated (guides, quickstart, postman collections, tutorial, fidesdemo, [database diagram](https://github.com/ethyca/fidesops/blob/main/docs/fidesops/docs/development/update_erd_diagram.md).

# Ticket

Fixes #502

### CODE REVIEW

**Review by eastandwestwind — CHANGES_REQUESTED** [review]
Just 1 suggestion, thanks for this @conceptualshark !

File: docs/fidesops/docs/guides/configuration_reference.md [hunk 1]
```diff
@@ -116,7 +116,7 @@ Please note: The configuration is case-sensitive, so the variables must be speci
 | `FIDESOPS__HOT_RELOAD` | False | If "True", the fidesops server will reload code changes without you needing to restart the server. This variable should always be set to "False" in production systems.|
 | `FIDESOPS__DEV_MODE` | False | If "True", the fidesops server will log error tracebacks, and log details of third party requests. This variable should always be set to "False" in production systems.|
 | `FIDESOPS__CONFIG_PATH` | None | If this variable is set to a path, that path will be used to load .toml files first. That is, any .toml files on this path will override any installed .toml files. |
-| `FIDESOPS__DATABASE__SQLALCHEMY_DATABASE_URI` | None | An optional override for the URI used for the database connection. |
+| `FIDESOPS__DATABASE__SQLALCHEMY_DATABASE_URI` | None | An optional override for the URI used for the database connection, in the form of `postgresql://<user>:<password>@<hostname>/<database>`. |
```
[the remark this answers is not in the document]
Comment (conceptualshark) [reply docs/fidesops/docs/guides/configuration_reference.md]: 👍 thank you!

**Review by eastandwestwind — APPROVED** [review]

**Comment by eastandwestwind** [comment]
Oh, forgot 1 thing, @conceptualshark do you think we should add to the `### Docs` section of `Changelog.md`?

**Comment by conceptualshark** [comment]
I wasn't sure if small updates like this should be included; the config option is already in the docs, this is just an actual example.

**Pull request merged**